### ISSUE TITLE
Change "notify" interface

### ISSUE THREAD

**Issue by boonebgorges** [livinglab/webwork-for-wordpress#155]
> Remove bell icon and replace with text link: ‘Follow’ with tooltip saying “Notify me of future replies to this question.”

**Comment by boonebgorges** [comment]
What do we show for users who have clicked the button? The text "unfollow" with a corresponding tooltip?

**Comment by jennaspevack** [comment]
How about this? For tooltip text: "Notify me of future replies" and "Stop notifications"
![follow-plus](https://user-images.githubusercontent.com/22797200/56986602-247ddb00-6b59-11e9-972f-aeddb7364554.png)
![following-check](https://user-images.githubusercontent.com/22797200/56986608-2a73bc00-6b59-11e9-9408-1d456ff05bf7.png)
![unfollow-minus](https://user-images.githubusercontent.com/22797200/56986613-2cd61600-6b59-11e9-85c1-40676347feb2.png)

**Comment by boonebgorges** [comment]
Thanks, @jennaspevack ! When is the "Following" text to be used? Only as a success message when initially clicking 'Follow'? (Subsequent pageloads then show 'Unfollow'?)

**Comment by jennaspevack** [comment]
Oops. Sorry that wasn't clear. I was thinking that unfollow would be a hover on the Following state.

Base state: FOLLOW / Tool Tip: "Notify me of future replies"
On-Click: Change to FOLLOWING
FOLLOWING State Hover: UNFOLLOW / ToolTip: "Stop notifications"

If this too complicated, then maybe lose the "Stop notifications" tooltip.

**Comment by boonebgorges** [comment]
Oh, gotcha! That makes more sense, and shouldn't be a problem to implement.

**Comment by boonebgorges** [comment]
I think I've captured it. Here's a gif (apologies, wanted to show hover!!):
![Screenshot_2019-05-01_13-39-07](https://user-images.githubusercontent.com/246627/57036510-1a6fe100-6c1a-11e9-8339-d10919362a0e.png)
![Peek 2019-05-01 14-03](https://user-images.githubusercontent.com/246627/57036517-20fe5880-6c1a-11e9-8625-19edcdd888b6.gif)

I made a judgment call about colors, based on what I saw in your mockup:
- When *subscribed*, color is blue. Hover is black.
- When *unsubscribed*, color is black. Hover is blue.
